README.md

```diff
@@ -0,0 +1,6 @@
+# Sysroots
+Contains the repos for both the MLAPI SDK and the UTP transport library
+
+Contents:
+* com.unity.multiplayer.mlapi -- the MLAPI SDK
+* com.unity.multiplayer.transport.utp -- the UTP transport library
```

Comment:
nit: this isn't the UTP library - just a wrapper/interface to it :)

Reply:
fixing